probatus/feature_elimination/feature_elimination.py

```diff
@@ -554,6 +559,11 @@ def fit(self, X, y, sample_weight=None, columns_to_keep=None, column_names=None,
             )
             remaining_features = list(set(current_features_set) - set(features_to_remove))
 
+            if features_to_remove:
+                idx = [features.index(feat) for feat in features_to_remove]
+                support_[idx] = False
```

Comment:
Isn't this going to set to False almost all the features?
As i understand this variable, we should get the support mask for the "optimal" feature set, so the one with minimum mean-std evaluation metric right? Currently, it is going to have support=True only for variables at which RFE stopped.